packages/flutter/lib/src/material/switch.dart

```diff
@@ -134,15 +134,21 @@ class Switch extends StatelessWidget {
   /// or macOS, following Material design's
   /// [Cross-platform guidelines](https://material.io/design/platform-guidance/cross-platform-adaptation.html).
   ///
-  /// On iOS and macOS, this constructor creates a [CupertinoSwitch], which has
-  /// matching functionality and presentation as Material switches, and are the
+  /// On iOS and macOS, this constructor creates a [Switch] which has matching
```

Comment:
```suggestion
  /// Creates a switch that looks and feels native when then the [ThemeData.platform] is iOS or macOS, otherwise a Material Design switch is created.
```